docs/framework/wcf/samples/jsonp.md

```diff
@@ -74,6 +74,6 @@ Sys._json0({"__type":"Customer:#Microsoft.Samples.Jsonp","Address":"1 Example Wa
   
 1.  Open the solution for the JSONP Sample.  
   
-2.  Press F5 to launch  HYPERLINK "http://localhost:26648/JSONPClientPage.aspx" http://localhost:26648/JSONPClientPage.aspx in the browser.  
+2.  Press F5 to launch  "[http://localhost:26648/JSONPClientPage.aspx]( http://localhost:26648/JSONPClientPage.aspx)" in the browser.  
```

Comment:
This was using an incorrect syntax for the link, which rendered as a literal `HYPERLINK "http....`, so I changed it to markdown link